hc32f4a2/ChangeLog.md

```
@@ -0,0 +1,1410 @@
+# Update History
+------
+## V2.3.0  Nov 08, 2024
+#### documents
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)